includes/misc-functions.php

```diff
@@ -1051,6 +1051,23 @@ function edd_is_promo_active() {
 	return false;
 }
 
+/**
+ * Gets the date that this EDD install was activated (for new installs).
+ * For existing installs, this option is added whenever the function is first used.
+ *
+ * @since 2.11.4
+ * @return string The timestamp when EDD was activated.
```

Comment:
```suggestion
 * @return int The timestamp when EDD was activated.
```

`time()` will be an integer.